tests/conftest.py

```diff
@@ -135,14 +135,16 @@
     "cloud",
     "docs",
     "filesystem",
-    "google_creds",
+    "mssql",
     "mysql",
     "openpyxl",
+    "performance",
```

Comment:
I updated the `google_creds` marker to `performance`. Since that is the only marker we exclude from our test run, I updated the coverage error message to explicitly state this. I also though `performance` conveyed this better since we performance tests are a different thing than correctness tests.

Reply:
This makes a lot of sense. Thank you